data_structures/Recursion/Count Zeros.py

```diff
@@ -0,0 +1,42 @@
+"""
+--------------------------------------   Count Zeros  -------------------------------------------
+
+Given an integer N, count and return the number of zeros that are present in the given integer using recursion.
+
+#### Input Format :
+    Integer N
+
+#### Output Format :
+    Number of zeros in N
+
+#### Constraints :
+    0 <= N <= 10^9
+
+#### Sample Input 1 :
+    10204
+#### Sample Output 1 :
+    2
+
+#### Sample Input 2 :
+    708000
+#### Sample Output 2 :
+    4
+"""
+
+def countZeros(n): 
+    if n<0: 
```

Comment:
you can have an space between n < 10 as in the next line you have n == 0 with spaces ;)